proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/chat/session/SessionCommandHandler.java

```diff
@@ -42,8 +42,8 @@ public Class<SessionPlayerCommand> packetClass() {
   public void handlePlayerCommandInternal(SessionPlayerCommand packet) {
     queueCommandResult(this.server, this.player, event -> {
       CommandExecuteEvent.CommandResult result = event.getResult();
-      if (!result.isAllowed()) {
-        if (!packet.argumentSignatures.isEmpty()) {
+      if (result == CommandExecuteEvent.CommandResult.denied()) {
```

Comment:
Pretty sure that this check needs to stay as-is, at least in regards to 1.19.x existing for now

Reply:
Forward but modifying the command has been handled below, error message is also deny.

Reply:
ah, I erm... left my glasses somewhere around here